src/ixdat/plotters/base_mpl_plotter.py

```diff
@@ -40,10 +50,14 @@ def new_two_panel_axes(self, n_bottom=1, n_top=1, emphasis="top"):
             n_bottom (int): 1 for a single y-axis, 2 for left and right y-axes on bottom
             emphasis (str or None): "top" for bigger top panel, "bottom" for bigger
                 bottom panel, None for equal-sized panels
+            interactive (bool): Whether to activate interactive range selection (default
+                True)
 
         Returns list of axes: top left, bottom left(, top right, bottom right)
         """
-        self.new_ax()  # necessary to avoid deleting an open figure, I don't know why
+        # Necessary to avoid deleting an open figure, I don't know why
```

Comment:
Ah, got it! new_ax() got called anyway for a reason I didn't understand before. But now it has a use :)